src/rebar3_format.erl

```diff
@@ -1,9 +1,23 @@
 %%% @doc Main entry point for the rebar3 format plugin
 -module(rebar3_format).
 
--export([init/1]).
+-elvis([{elvis_style, no_debug_call, disable}]).
+
+-hank([{unnecessary_function_arguments, [{main, 1, 1}]}]).
+
+-export([init/1, main/1]).
 
 %% @private
 -spec init(rebar_state:t()) -> {ok, rebar_state:t()}.
 init(State) ->
     rebar3_format_prv:init(State).
+
+%% @private
+-spec main(any()) -> no_return().
+main(_) ->
+    Msg = "rebar3_format shall be run as a plugin, not as an escript.~n",
+
+    % Not possible (no other module found in an escript context):
+    %rebar_api:abort(Msg, []).
```

Comment:
Since this will run outside rebar3's context… I guess we can remove these comments.

Reply:
Sure, this is done (commit https://github.com/AdRoll/rebar3_format/pull/332/commits/f3fb4b802184bd3c6e480d6c26aa1b203e44e1f8).